test/ssc_test/cmod_battery_pvsamv1_test.cpp

```diff
@@ -383,7 +388,12 @@ TEST_F(CMPvsamv1BatteryIntegration_cmod_pvsamv1, ResidentialDCBatteryModelIntegr
             pairs["batt_dispatch_wf_forecast_choice"] = 0;
             pairs["batt_dispatch_load_forecast_choice"] = 0;
             break;
-
+        case 3:
+            // Input grid power targets
```

Comment:
Update comment to match the correct name, please.

Reply:
Done